src/reactNativeUtils.js

```diff
@@ -101,3 +101,60 @@ export function androidRelease(): string {
   // (If changing the implementation, adjust comment below jsdoc.)
   return (Platform.constants.Release: string);
 }
+
+/**
+ * The manufacturer of the Android device.
+ *
+ * E.g., "Google", "samsung".
+ */
+// This is Build.MANUFACTURER as of RN v0.68.7:
+//   https://reactnative.dev/docs/platform#constants
+//   https://github.com/facebook/react-native/blob/v0.68.7/ReactAndroid/src/main/java/com/facebook/react/modules/systeminfo/AndroidInfoModule.java#L73
+//   https://developer.android.com/reference/android/os/Build#MANUFACTURER
+export function androidManufacturer(): string {
+  invariant(Platform.OS === 'android', 'androidRelease called on iOS');
+
+  // Flow isn't refining `Platform` to a type that corresponds to values
+  // we'll see on Android.
+  //
+  // (If changing the implementation, adjust comment below jsdoc.)
+  return (Platform.constants.Manufacturer: string);
+}
+
+/**
+ * "The consumer-visible brand with which the product/hardware will be associated, if any."
+ *
+ * E.g., "google", "samsung".
+ */
+// This is Build.BRAND as of RN v0.68.7:
+//   https://reactnative.dev/docs/platform#constants
+//   https://github.com/facebook/react-native/blob/v0.68.7/ReactAndroid/src/main/java/com/facebook/react/modules/systeminfo/AndroidInfoModule.java#L74
+//   https://developer.android.com/reference/android/os/Build#BRAND
+export function androidBrand(): string {
+  invariant(Platform.OS === 'android', 'androidRelease called on iOS');
+
+  // Flow isn't refining `Platform` to a type that corresponds to values
+  // we'll see on Android.
+  //
+  // (If changing the implementation, adjust comment below jsdoc.)
+  return (Platform.constants.Brand: string);
+}
+
+/**
+ * "The end-user-visible name for the end product."
+ *
+ * E.g., "sdk_gphone64_x86_64", "SM-G960U1".
```

Comment:
> Unfortunately, in my test using an emulated Pixel XL (results recorded in jsdoc), none of these three fields actually had anything like "Pixel XL" in it.

On my physical Pixel 5, I get:
```
  "manufacturer": "Google",
  "brand": "google",
  "model": "Pixel 5",
```

So that's encouraging. And for this line we can write:

```suggestion
 * E.g., "Pixel 5", "sdk_gphone64_x86_64", "SM-G960U1".
```